db/migrate/20220316112023_change_membership_can_manage_locations.rb

```diff
@@ -0,0 +1,7 @@
+class ChangeMembershipCanManageLocations < ActiveRecord::Migration[6.1]
```

Comment:
I know that this was done in the past, but this is probably better done using SQL - reason is that if we ever rename the Membership model or one of the attributes this will stop working. See: https://guides.rubyonrails.org/v3.2/migrations.html#using-models-in-your-migrations.

Reply:
Indeed, I've written the migration just using SQL now.